.github/workflows/Linux_x86_64.yml

```diff
@@ -0,0 +1,48 @@
+name: Linux_x86_64
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
its important that linux_x86_64 is build on debian:stretch-backports as that provides libs old enough give the releases broad comparability across the Linux ecosystem. I only just now realize that you moved it to debian:testing on CircleCI already, but we will need to revert that. Sorry for not realizing this during the previous pr.

Reply:
Only the x64, not the x86 or sdl1?

Reply:
Perhaps that should come as a separate PR following this one?

Reply:
yeah my only concern there is that its easier to revert as is...

Reply:
> Only the x64, not the x86 or sdl1?

linux_x86_64 is the only one that we produce releases for, x86 and sdl1 are more for self build targets or crossbuild from modern compilers, so its fine to run them on recent distros.

Reply:
#1846

Reply:
I just tried to go back to debian:stretch-backports, but libfmt-dev doesn't exist in the debian stable or stretch-backports repos.

Reply:
That shouldn't be an issue, we build fmt from source automatically if it doesn't exist